packages/pluggableWidgets/video-player-native/src/__tests__/__snapshots__/VideoPlayer.spec.tsx.snap

```diff
@@ -13,11 +13,21 @@ exports[`VideoPlayer hides the loading indicator after load 1`] = `
   }
 >
   <Video
+    accessible={true}
```

Comment:
I see you've updated the snapshot but there's no new unit tests for the newly introduced functionality:
- on android, introduce a modal and fullscreen behaviour + controls
- on ios, maintian previous functionality

please add these while it is fresh in your head

Reply:
done